core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -349,26 +349,25 @@ public Configuration defineConfiguration(String name, String template, Configura
          if (c == null) {
             throw log.undeclaredConfiguration(template, name);
          } else {
-            return defineConfiguration(name, false, configurationOverride, c);
+            return actualDefineConfiguration(name, configurationOverride, c);
```

Comment:
Yeah, I saw you didn't really change it, but I still assigned https://issues.jboss.org/browse/ISPN-7587 to you ;)